Frameworks/CoreText/CTFont.mm

```diff
@@ -901,20 +900,19 @@ CTFontRef CTFontCreateWithGraphicsFont(CGFontRef cgFont, CGFloat size, const CGA
     return (CTFontRef)ret;
 }
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+ @Notes options is not supported, and is deprecated anyway
```

Comment:
I was going off of the decision w/ CTFontCopyTable where we're still marking it Interoperable despite not supporting kCTFontTableOptionExcludeSynthetic